### PR TITLE
[Editorial] Maximum Building II

### PR DESCRIPTION
**Pull request by brebenelmihnea**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by brebenelmihnea** [comment]
https://cses.fi/problemset/result/9656578/
image

File: solutions/gold/cses-1148.mdx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+int main() {
+	int n, m;
+	cin >> n >> m;
+	vector<vector<int>> r(n + 2, vector<int>(m + 2));
+	vector<vector<int>> u(n + 2, vector<int>(m + 2));
+	vector<vector<int>> d(n + 2, vector<int>(m + 2));
+	vector<vector<int>> ans(n + 3, vector<int>(m + 3));
+	vector<vector<char>> mat(n + 2, vector<char>(m + 2));
+	for (int i = 1; i <= n; i++) {
```
Comment (SansPapyrus683) [inline solutions/gold/cses-1148.mdx:48]: are the +2's and +3's the exact size needed

**Review by envyaims — APPROVED** [review]
clownsans

File: solutions/gold/cses-1148.mdx (hunk 1 above)
Comment (brebenelmihnea) [reply solutions/gold/cses-1148.mdx:48]: yes

**Review by dongliu0426 — APPROVED** [review]

File: solutions/gold/cses-1148.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/gold/cses-1148.mdx:48]: hold on i'm looking at the access to `mat`
the max indices are to `mat[n][m]`, so +1 is the max one needed

**Review by ryanchou-dev — APPROVED** [review]

File: solutions/gold/cses-1148.mdx (hunk 1 above)
Comment (brebenelmihnea) [reply solutions/gold/cses-1148.mdx]: done

**Comment by SansPapyrus683** [comment]
image

1. sus formatting
2. use \texttt for `ans`

**Comment by SansPapyrus683** [comment]
for sus formatting i probably should've been a bit more specific
instead of using ++ maybe just say "increment [x] by 1"

**Review by SansPapyrus683 — APPROVED** [review]

**Comment by SansPapyrus683** [comment]
too dumb to actually know what's going on anyways

**Pull request merged**